zjit/src/hir.rs

```diff
@@ -886,7 +886,7 @@ impl<'a> std::fmt::Display for InsnPrinter<'a> {
     fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
         match &self.inner {
             Insn::Const { val } => { write!(f, "Const {}", val.print(self.ptr_map)) }
-            Insn::Param { idx } => { write!(f, "Param {idx}") }
```

Comment:
We maybe sacrifice a bit of debuggability by not being able to immediately see which param is which in the hir printout, but these get passed to BB params very quick anyways so it's probably fine